File: 2525. Categorize Box According to Criteria.py
class Solution:
    def categorizeBox(self, length: int, width: int, height: int, mass: int) -> str:
        dim, mas = '', ''

        vol = length * width * height
        if length >= 10**4 or width >= 10**4 or height >= 10**4 or vol >= 10**9:
            dim = "Bulky"
        if mass >= 100:
            mas = "Heavy"

        if dim == "Bulky" and mas == "Heavy":
            rst = 'Both'
        if dim == "Bulky" and mas != "Heavy":
            rst = 'Bulky'
        if dim != "Bulky" and mas == "Heavy":
            rst = 'Heavy'
        if dim != "Bulky" and mas != "Heavy":
            rst = 'Neither'
        return rst

File: c++/2525. Categorize Box According to Criteria.cpp
/*

*/

class Solution {
public:
    string categorizeBox(int l, int w, int h, int m) {
        long long v=(long long)l*(long long)w*(long long)h;
        if(l>=10000||w>=10000||h>=10000||v>=pow(10,9)){
            if (m>=100) return "Both";
            else return "Bulky";
        }
        else if (m>=100) return "Heavy";
        else return "Neither";
    }
};
